### PR TITLE
Add VTK Renderer support to Endoscopy Tool Tracking Python app

### PR DESCRIPTION
**Pull request by mocsharp**
This PR add Python bindings to the VTK operator and adds VTK renderer support to the Endoscopy Tool Tracking Python app.

DO NOT merge before #412

### CODE REVIEW

**Review by grlee77 — CHANGES_REQUESTED** [review]
Thanks, it looks good.

please look into potentially allowing this to work for older versions as well per suggestions in #412

File: applications/endoscopy_tool_tracking/python/metadata.json [hunk 1]
```diff
@@ -10,12 +10,13 @@
 		"language": "Python",
 		"version": "1.0",
 		"changelog": {
-			"1.0": "Initial Release"
+			"1.0": "Initial Release",
+			"2.0": "Support VTK renderer"
 		},
 		"holoscan_sdk": {
-			"minimum_required_version": "0.5.0",
+			"minimum_required_version": "2.1.0",
```
[the remark this answers is not in the document]
Comment (tbirdso) [reply applications/endoscopy_tool_tracking/python/metadata.json:17]: Sorry I didn't see this comment. Yes, `dev_container build_and_run` is not tied to a specific Holoscan SDK version and does support v0.6.

File: applications/endoscopy_tool_tracking/python/endoscopy_tool_tracking.py [hunk 2]
```diff
@@ -37,6 +37,9 @@
 # from holohub.qcap_source import QCAPSourceOp
 from holohub.tool_tracking_postprocessor import ToolTrackingPostprocessorOp
 
+# Enable this line for vtk rendering
+# from holohub.vtk_renderer import VtkRendererOp
```
Comment (tbirdso) [inline applications/endoscopy_tool_tracking/python/endoscopy_tool_tracking.py:41]: Could you please also mention lines to uncomment in the README?

Comment (mocsharp) [reply applications/endoscopy_tool_tracking/python/endoscopy_tool_tracking.py:41]: Done! Updated README. :pray:

**Review by grlee77 — APPROVED** [review]
Thanks @mocsharp, approved!

**Review by tbirdso — APPROVED** [review]
Thanks @mocsharp . Looks like there is one merge conflict caused by https://github.com/nvidia-holoscan/holohub/pull/412 to resolve.

**Pull request merged**